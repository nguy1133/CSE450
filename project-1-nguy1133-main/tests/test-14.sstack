// PUSH's can be intermixed with other commands.
PUSH "1st"
PUSH "2nd"
PRINT
PUSH "3rd"
PRINT
PRINT
