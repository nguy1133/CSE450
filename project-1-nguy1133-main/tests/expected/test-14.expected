2nd
3rd
1st
